frontend_build/src/read_bundle_plugins.js

```diff
@@ -24,8 +24,6 @@ var libs = function(kolibri_name) {
     'logging': kolibri_name + '.lib.logging',
     'vue': kolibri_name + '.lib.vue',
     'kolibri': kolibri_name,
-    'core-base': kolibri_name + '.lib.coreBase',
```

Comment:
The alias stuff is necessary to allow any test code to build properly - these ones make the mapping for production/dev environments to be able to reach into the core app and access code.

I agree it could all be managed in a neater way, just still not sure how currently!